hal/hal_i2c.h

```diff
@@ -77,7 +90,8 @@ void i2c_write_blocking(const i2c_periph_inst_t* I2C_instance, const unsigned ch
  * @param stop_bit Does this transaction end with or without a stop-bit: Value 1 is with stop-bit
```

Comment:
Leave it as @param but I suggest you also add some wiki-style comment above these calls which helps to get a more global understanding of the (im)possibilities of hal_i2c.h and possibly do so with the other key files as well.